scene/3d/skeleton_3d.cpp

```diff
@@ -95,17 +95,32 @@ bool Skeleton3D::_set(const StringName &p_path, const Variant &p_value) {
 	} else if (what == "scale") {
 		set_bone_pose_scale(which, p_value);
 #ifndef DISABLE_DEPRECATED
-	} else if (what == "pose") {
+	} else if (what == "pose" || what == "bound_children") {
```

Comment:
I'd put this in two statements and duplicate the message instead to make it clearer and avoid duplicate check for the name

Reply:
The reason it’s like this is because the check for “bound_children” is effectively a “pose” check (we don’t do anything with the property itself, we just use it to set the pose), and we don’t want to print the deprecated message twice for the same resource.

Reply:
Are they always set together? Then maybe just print it for one?

Reply:
In some cases, but often the whole skeleton is set to the rest pose and would not have the “pose” property on any of the bones. In that case, “bound_children” is still an indicator that it has the old skeleton format with the old “pose” format, so the message would still apply.